Dockerfile

```diff
@@ -1,25 +1,18 @@
 # Openhim Console Dockerfile for latest changes
-FROM ubuntu:16.04
+FROM node:14.17-alpine as build
+RUN apk upgrade --update-cache --available
```

Comment:
Is this step necessary?

Reply:
I had added it to make sure we are not missing any required dependencies. Let me test and see what's happening

Reply:
Looks good so far.